models/intel/pedestrian-and-vehicle-detector-adas-0001/description/pedestrian-and-vehicle-detector-adas-0001.md

```diff
@@ -21,7 +21,7 @@ Pedestrian and vehicle detection network based on MobileNet v1.0 + SSD.
 | Source framework                | Caffe*                |
 
 Average Precision (AP) metric is described in: Mark Everingham et al.
-[The PASCAL Visual Object Classes (VOC) Challenge](http://host.robots.ox.ac.uk/pascal/VOC/pubs/everingham10.pdf).
+[The PASCAL Visual Object Classes (VOC) Challenge](https://link.springer.com/content/pdf/10.1007/s11263-009-0275-4.pdf).
```

Comment:
This link doesn't work for anyone without a Springer subscription. Moreover, this change has nothing to do with the topic of the PR. Please revert it.

Reply:
I've changed it because paper was moved, so documentation tests failed because of broken link. But okey, I'll revert it

Reply:
FYI, it wasn't moved, `host.robots.ox.ac.uk` is just a very unstable website which keeps going down. Just ignore it.

Reply:
Oh, I didn't know it. Will keep it in mind, thank you